flink-filesystems/flink-s3-fs-base/pom.xml

```diff
@@ -161,12 +161,17 @@ under the License.
 									<pattern>org.apache.flink.runtime.fs.hdfs</pattern>
```

Comment:
Maybe we could place `util` under `fs` and then relocate `o.a.f.runtime.fs`. This would save us the second relocation pattern.